src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+  caption="Real-user Cummulative Layout Shift experiences.",
+  description='Bar chart showing the top 5 CMSs and whether they have a "good" First Input Delay experience. Wordpress has a 47% of desktop sites with a "good experience" and 57% of mobile sites. Drupal has 58% for desktop and 70% for mobile, Joomla has 51% for desktop and 63% for mobile, Squarespace has 35% for desktop and 44% for mobile, and Wix has 58% for desktop and 59% for mobile.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=1265001868&format=interactive",
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+The top 5 CMSs do poorly here. Only 50% of web pages loaded by a top 5 CMS have a “good” CLS experience, with this figure rising to 59% on mobile. Across all CMSs the average desktop score is 59% and average mobile score is 67%. This shows us all CMSs have work to do here, but the top 5 CMSs in particular need improvement. 
+
+**Lighthouse scores**
+
+[Lighthouse](https://almanac.httparchive.org/en/2019/methodology#lighthouse) is an open-source, automated tool designed to help developers assess and improve the quality of their websites. One key aspect of the tool is that it provides a set of audits to assess the status of a website in terms of performance, accessibility, progressive web apps, and more. For the purposes of this chapter, we are interested in two specific audits categories: PWA and accessibility.
+
+**PWA**
+
+The term [Progressive Web App (PWA)](https://web.dev/progressive-web-apps/) refers to web-based user experiences that are considered as being reliable, fast, and engaging. Lighthouse provides a set of audits which returns a PWA score between 0 (worst) and 1 (best). These audits are based on the [Baseline PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist#baseline), which lists 14 requirements. Lighthouse has automated audits for 11 of the 14 requirements. The remaining 3 can only be tested manually. Each of the 11 automated PWA audits are weighted equally, so each one contributes approximately 9 points to your PWA score.
+
+{{ figure_markup(
+  image="cms-pwa-compliance-lighthouse-score.png",
+  caption="PWA compliance.",
+  description="Bar chart showing the median PWA compliance score for each of the top 5 cmss. Wordpress has a median score of 32% and the rest (Joomla, Drupal, Wix and Squarespace) all have 29%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=904129607&format=interactive",
+  sheets_gid="1497775920",
+  sql_file="median_lighthouse_score.sql"
+  )
+}}
+
+The median Lighthouse PWA score is 29% for the top 5 CMSs, with the exception of WordPress which comes in at 32%. Across all CMSs, the average median score is 34%, with a minimum of 4% and a maximum of 71%.
```

Comment:
Whenever possible, add as much context as you can about what these numbers mean and how readers should feel about them. Any ideas why WordPress performs better than other CMSs?

Reply:
This feels way outside my area of expertise – I'm not a developer so I have no idea here